source/style/content.css

```diff
@@ -33,6 +33,19 @@ body {
 	border: 0 !important;
 }
 
+/* Highlight Mention in stream */
+[data-mentioned-user-id] {
+    position: relative;
+    padding: 2px 5px 4px 5px;
+}
+
+.tweet.mention4me {
+    background-color: rgba(var(--bgcolor-values), 0.1);
```

Comment:
The variable name here should also reflect the namespace I mentioned on `user-choice-color.js`